pkg/sync/v9/data/ClusterRole.rbac.authorization.k8s.io/strimzi-cluster-operator-namespaced.yaml

```diff
@@ -0,0 +1,268 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  labels:
+    app: strimzi
+  name: strimzi-cluster-operator-namespaced
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts
```

Comment:
Not complaining, but this SA is powerful :/

Reply:
Yeah, I have just used what they have defined :man_shrugging: